### PR TITLE
Fix REST API tests

### PR DESCRIPTION
**Pull request by Marishka17**
### Motivation and context

Fixed REST API tests after merging #5408 and #5396 to develop

### CODE REVIEW

File: tests/python/shared/fixtures/data.py [hunk 1]
```diff
@@ -357,8 +357,17 @@ def find(jobs):
...
-        return list(filter(lambda t: annotations["task"][str(t["id"])]["shapes"], tasks))
+    def find(tasks, skip_tasks=None):
+        def _is_task_with_shapes(t):
+            if (
+                skip_tasks is not None
+                and t["id"] in skip_tasks
+                or not annotations["task"][str(t["id"])]["shapes"]
+            ):
+                return False
+            return True
```
Comment (kirill-sizov) [inline tests/python/shared/fixtures/data.py:368]: Can we use such solution?
```suggestion
    def find(tasks):
        def _is_task_with_shapes(t):
            if (
                not t.get("data")
                or not annotations["task"][str(t["id"])]["shapes"]
            ):
                return False
            return True
```

Comment (Marishka17) [reply tests/python/shared/fixtures/data.py]: ok, thanks

**Comment by kirill-sizov** [comment]
@bsekachev Are these e2e tests unstable? Or are they related with this PR?

**Comment by bsekachev** [comment]
@kirill-sizov 

case_82_canvas3d_functionality_cuboid_opacity_outlined_borders.js

This one was fixed: https://github.com/opencv/cvat/pull/5409#issuecomment-1334340257

case_118_multi_tasks.js 

Was quite stable before, not sure why it is failed here.

**Review by kirill-sizov — APPROVED** [review]
LGTM

**Review by nmanovic — APPROVED** [review]

**Pull request merged**